pages/trainer.page.php

```diff
@@ -76,19 +76,19 @@
 			<thead>
 			<tr>
 				<th>#</th>
-				<th>Same Level</th>
-				<th>Name</th>
-				<th>Level</th>
-				<th>Gyms</th>
-				<th>Last Seen</th>
+				<th><?= $locales->TRAINERS_TABLE_SAME_LEVEL ?></th>
+				<th><?= $locales->TRAINERS_TABLE_NAME ?></th>
+				<th><?= $locales->TRAINERS_TABLE_LEVEL ?></th>
+				<th><?= $locales->TRAINERS_TABLE_GYMS ?></th>
+				<th><?= $locales->TRAINERS_TABLE_LAST_SEEN ?></th>
 			</tr>
 			</thead>
 			<tbody id="trainersContainer">
 
 			</tbody>
 			<tfoot>
 				<tr class="loadMore text-center">
-					<td colspan="6"><button id="loadMoreButton" class="btn btn-default hidden"><?= $locales->TRAINER_LOAD_MORE ?></button></td>
+					<td colspan="6"><button id="loadMoreButton" class="btn btn-default hidden"><?= $locales->TRAINERS_LOAD_MORE ?></button></td>
```

Comment:
Please change the key to TRAINER**S**_LOAD_MORE in translation.json for all languages.